python/mxnet/optimizer/lars.py

```diff
@@ -252,6 +253,13 @@ def fused_step(self, indices, weights, grads, states):
                 wd = wds[i]
                 lr = lrs[i]
                 lr *= self._get_lars(index, weight, grad, wd)
+                # normal SGD picks up momentum correction by default,
+                # so need to modify the momentum to undo that.
+                # The correction term is previous_lr / current_lr.
+                kwargs['momentum'] = (self.momentum * (self.last_lr / lr)) \
```

Comment:
Thank you for the fix!

It will reduce the times of getting an item of `self.last_lr`.
```python
kwargs['momentum'] = (self.momentum * (self.last_lr.get(index, lr) / lr)) \
                                     if lr != 0 else \
                                     self.momentum
```

Reply:
Thanks, done